manifests/maintenance/pg_repack.pp

```diff
@@ -63,10 +60,31 @@
     command  => "${repack} ${repack_jobs} ${other_tables} ${logging}",
   }
 
-  cron { 'pg_repack reports tables' :
-    monthday => 10,
-    hour     => 5,
-    minute   => 30,
-    command  => "${repack} ${repack_jobs} ${reports_tables} ${logging}",
+  if versioncmp($facts['pe_server_version'], '2019.7.0') == -1 {
+    cron { 'pg_repack reports tables' :
+      monthday => 10,
+      hour     => 5,
+      minute   => 30,
+      command  => "${repack} ${repack_jobs} ${reports_table} ${logging}",
+    }
+  }
+  else {
+    cron { 'pg_repack reports tables' :
+      ensure   => 'absent',
+    }
+  }
+
+  if versioncmp($facts['pe_server_version'], '2019.3.0') == -1 {
+    cron { 'pg_repack resource_events tables' :
```

Comment:
I changed the start time for one of the crons, because I'm not too sure of a good way to combine them into the same cron when using the versioncmp function.